test_files/module/main0.d

```diff
@@ -0,0 +1,7 @@
+module modules.main0;
+
+public import modules.include;
```

Comment:
Is it necessary for imports to be public if we're using the symbols as a base for adding imports instead of the includes?

I'm not sure if we actually want to exactly map the C headers in D. Public imports is usually not idiomatic D, unless it's in a package module.